ppd/ppd-generator.c

```diff
@@ -610,39 +610,39 @@ ppdCreatePPDFromIPP2(char         *buffer,          // I - Filename buffer
     int resArray[__INT16_MAX__]; // Creating a resolution array supporting a maximum of 32767 resolutions.
     int lowdpi=0, middpi=0, hidpi=0; //Lower , middle and higher resolution
     if ((attr = ippFindAttribute(response, "urf-supported",
-				 IPP_TAG_KEYWORD)) != NULL)
+			IPP_TAG_KEYWORD)) != NULL)
     {
-       for (int i = 0, count = ippGetCount(attr); i < count; i ++)
-      {
-	     const char *rs = ippGetString(attr, i, NULL); // RS values
-       const char *rsCopy=ippGetString(attr, i, NULL); // RS values(copy) 
- 	     if (strncasecmp(rs, "RS", 2)) // Comparing attributes to have RS in the beginning to indicate the resolution feature
+      for (int i = 0, count = ippGetCount(attr); i < count; i ++)
+    {
+	      const char *rs = ippGetString(attr, i, NULL); // RS values
+        const char *rsCopy=ippGetString(attr, i, NULL); // RS values(copy) 
+ 	      if (strncasecmp(rs, "RS", 2)) // Comparing attributes to have RS in the beginning to indicate the resolution feature
 	        continue;
-       int resCount=0;// Using a count variable which can be reset 
-       while (*rsCopy != '\0')// Parsing through the copy pointer to determine the no. of resolutions
-        {
+        int resCount=0;// Using a count variable which can be reset 
+        while (*rsCopy != '\0')// Parsing through the copy pointer to determine the no. of resolutions
+         {
           if(*rsCopy=='-')
           {
             resCount++;
           }
           rsCopy++;
-        }
+         }
         resCount++;
         resStore=resCount;
         resCount=0;
         resArray[resCount]=atoi(rs+2);
         resCount++;
         while (*rs != '\0') // Parsing through the entire pointer and appending each resolution to an array
         {
-           if(*rs=='-')
-           {
+          if(*rs=='-')
```

Comment:
Could you also check whether you have spaces around operators and between `if`, `while`, ... and the opening braces:
```
if (*rs == '-')
```
Could you check through all your code for this?